source/views/menus/menu-bonapp.js

```diff
@@ -83,19 +86,28 @@ export class BonAppHostedMenu extends React.PureComponent<Props, State> {
     }
   }
 
+  requestMenu = (cafeId: string) => () =>
+    fetchJsonQuery(bonappMenuBaseUrl, {cafe: cafeId})
+  requestCafe = (cafeId: string) => () =>
+    fetchJsonQuery(bonappCafeBaseUrl, {cafe: cafeId})
+
   fetchData = async (props: Props) => {
     let cafeMenu: ?MenuInfoType = null
     let cafeInfo: ?CafeInfoType = null
 
     try {
       ;[cafeMenu, cafeInfo] = await Promise.all([
-        fetchJsonQuery(bonappMenuBaseUrl, {cafe: props.cafeId}),
-        fetchJsonQuery(bonappCafeBaseUrl, {cafe: props.cafeId}),
+        retry(this.requestMenu(props.cafeId), {retries: 3}),
+        retry(this.requestCafe(props.cafeId), {retries: 3}),
       ])
     } catch (error) {
-      tracker.trackException(error.message)
-      bugsnag.notify(error)
-      this.setState(() => ({error}))
+      if (error.message === "JSON Parse error: Unrecognized token '<'") {
```

Comment:
Could there ever be any other type of unrecognized token other than `<` we'd want to check for? Perhaps the comparison could be for "JSON Parse error:"

Reply:
I know what `<` signifies. Other errors I would like to handle separately.

Reply:
Ok.